localization/replace/english/wc_mottos_l_english.yml

```diff
@@ -0,0 +1,43 @@
+﻿l_english:
+ # Norse-specific
+ motto_odins_wisdom:0 "Odyn's Wisdom"
+ motto_odins_brilliance:0 "Odyn's Brilliance"
+ motto_odins_frenzy:0 "Odyn's Frenzy"
+ motto_odins_sorcery:0 "Odyn's Sorcery"
+ motto_odins_eye:0 "Odyn's One Eye"
+ motto_odins_ravens:0 "Odyn's Ravens"
+ motto_friggs_wisdom:0 "Eonar's Wisdom"
+ motto_friggs_loins:0 "Eonar's Bounteous Loins"
```

Comment:
> What event?

I didn't screenshot the event but it is similar to Determine Personal Deity from base Northern Lords, here's the loc for it.

Reply:
> > What event?
> 
> I didn't screenshot the event but it is similar to Determine Personal Deity from base Northern Lords, here's the loc for it.

This localization is used in the mottos.
I checked and couldn't find working entrances of the game assigning Norse gods. That's why I asked about the event.

Reply:
I tried to search for an event ID but the best I got is another loc for it on ``decision_l_english.yml``:
```
select_personal_deity_germanic_decision:0 "Determine Personal Deity"
 select_personal_deity_germanic_decision_tooltip:0 "Think about [ROOT.Char.GetFaith.PantheonTerm]"
 select_personal_deity_germanic_decision_desc:1 "#F It is not uncommon for [ROOT.Char.GetFaith.GetAdherentNamePluralNoTooltip] to single out one amongst [ROOT.Char.GetFaith.PantheonTerm] who suits their particular tastes and aims in life for special attention.\n\nI should contemplate these divine beings and decide which one calls to me the most.#!"
 select_personal_deity_germanic_decision_confirm:0 "Dedicate your Worship to a particular God"
 select_personal_deity_germanic_decision_tt:1 "You begin to live your life according to the values of"
 select_personal_deity_germanic_decision.select_odin:0 "Odin, Glad-of-War, the All-Father"
 select_personal_deity_germanic_decision.select_ullr:0 "Ullr, Swiftest on Snow"
 select_personal_deity_germanic_decision.select_tyr:0 "Tyr, the One Handed"
 select_personal_deity_germanic_decision.select_thor:0 "Thor, Guardian of Man"
 select_personal_deity_germanic_decision.select_freyr:0 "Freyr, Bringer of Peace"
```

Reply:
> I tried to search for an event ID but the best I got is another loc for it on `decision_l_english.yml`:

I removed this event before, that's why I surprised you got a Norse god modifier.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113432785-1264e100-93ef-11eb-8f08-14e901c65017.png)

</details>

Reply:
Yeah it appears so that it has been commented. If I find that event again I'll identify the ID.